Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -720,11 +724,24 @@ private static IEnvironment GetEnvironment(IDeploymentSettingsManager settings =
             string root = PathResolver.ResolveRootPath();
             string siteRoot = Path.Combine(root, Constants.SiteFolder);
             string repositoryPath = Path.Combine(siteRoot, settings == null ? Constants.RepositoryPath : settings.GetRepositoryPath());
+            string binPath = HttpRuntime.BinDirectory;
+
+            if (!string.IsNullOrWhiteSpace(binPath) && !OSDetecter.IsOnWindows())
+            {
+                int binIdx = binPath.LastIndexOf("Bin", StringComparison.Ordinal);
```

Comment:
binPath is from "HttpRuntime.BinDirectory", it always pointing to Kudu`s bin path, always end with ..../Bin, unless we change Kudu`s folder structure.
why we need to worry about site name here?